libraries/AP_Compass/AP_Compass.cpp

```diff
@@ -15,6 +15,8 @@
 
 extern AP_HAL::HAL& hal;
 
+Compass Compass::frontend;
+
```

Comment:
well... if you are inside the class you don't even have to call get_frontend(). this is only for those outside of the class.